main/menu.ts

```diff
@@ -20,14 +20,28 @@ export default function setMenu(win: Electron.BrowserWindow) {
                     label: 'Undo',
                     accelerator: 'CmdOrCtrl+Z',
                     click: () => {
-                        win.webContents.send('nyaovim:exec-commands', ['undo']);
+                        const focusedWebContents = webContents.getFocusedWebContents();
+                        if (win.webContents === focusedWebContents) {
```

Comment:
Oh, it must be a bug of type definitions. I'll send a PR for it.

And you can add workaround as below:

```typescript
(win.webContents as any).isFocused()
```

The workaround can be removed after type definition is fixed. So could you add the workaround for now?